packages/cloud-cognitive/src/components/CreateTearsheet/CreateTearsheet.js

```diff
@@ -358,7 +371,6 @@ export let CreateTearsheet = forwardRef(
                   tearsheetSectionComponents.map(
                     (tearsheetSection, sectionIndex) => (
                       <SideNavLink
-                        href="javascript:void(0)"
```

Comment:
This was previously needed to get this component working from Carbon. I'm guessing Carbon made an update recently and we don't need this line anymore (in fact, it was breaking the scrolling behavior).

Reply:
Can you replace this href with `href={`#${childGoesHere.props.id}`}` and add an `ev.preventDefault()` to your onClick for `SideNavLink`?

Reply:
Updated!